src/documentation/ToolkitDocumentation/ToolkitDocumentation.tsx

```diff
@@ -169,7 +169,7 @@ const ActiveTooklitLevel = ({
         {toolkit.contents?.map((topic) => (
           <ToolkitTopLevelListItem
             key={topic.name}
-            name={topic.name}
+            name={topic.name + (isV2Only(topic) ? " (V2)" : "")}
```

Comment:
This and similar below are temporary, we'll soon revisit this to add the purple tag in the design and perhaps unify the logic.